src/WebJobs.Script/Rpc/FunctionRegistration/FunctionDispatcher.cs

```diff
@@ -275,6 +275,26 @@ private async Task RestartWorkerChannel(string runtime, string workerId)
             }
         }
 
+        private void ModifyErrorBucketAndAdd(WorkerErrorEvent currentErrorEvent, TimeSpan threshold = default)
+        {
+            if (threshold == default)
+            {
+                threshold = TimeSpan.FromSeconds(10);
```

Comment:
Process startup times could vary from few seconds to minutes. We would like ensure that language worker is healthy and is executing invocations for sometime. Threshold of 30mins sounds good. Also, add this is a constant to LanguageWorkerConstants.cs